services/QuillLMS/client/app/modules/pusherInitializer.ts

```diff
@@ -0,0 +1,21 @@
+import Pusher from 'pusher-js';
+
+const pusherInitializer = (id: number, channelEvent: string, callback?: () => void) => {
```

Comment:
This is a really helpful module to have on hand.

Reply:
I think there's probably a way to make this a custom hook but I ran out of time.